torchvision/prototype/transforms/_augment.py

```diff
@@ -86,15 +86,19 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
         return dict(i=i, j=j, h=h, w=w, v=v)
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
-        if isinstance(inpt, features._Feature):
-            return inpt.erase(**params)
+        if isinstance(inpt, (features.Image, torch.Tensor)):
+            output = F.erase_image_tensor(inpt, **params)
+            if isinstance(inpt, features.Image):
+                return features.Image.new_like(inpt, output)
+            return output
         elif isinstance(inpt, PIL.Image.Image):
             # TODO: We should implement a fallback to tensor, like gaussian_blur etc
             raise RuntimeError("Not implemented")
         elif isinstance(inpt, torch.Tensor):
             return F.erase_image_tensor(inpt, **params)
-        else:
-            return inpt
+        raise TypeError(
```

Comment:
This means we no longer support meta-data such as integers, strings etc. Where else did we make this change? We probably need to undo it. Here we should check for bboxes and masks and throw an error. Labels should be permitted and returned as-is.